Dockerfile

```diff
@@ -48,3 +48,12 @@ RUN rustup component add \
   rls \
   rust-analysis \
   rust-src
+
+# Install embedmd (via Go)
+ARG GOLANG_VERSION=1.13.1
+ENV GOROOT /usr/local/go
+ENV GOPATH $HOME/go
+ENV PATH $GOROOT/bin:$GOPATH/bin:$PATH
+RUN mkdir -p ${GOROOT}
```

Comment:
(nit) We seem to not put variables in {} in this file, unless necessary (and below)

```suggestion
RUN mkdir -p $GOROOT
```

Reply:
Done.